PocketKit/Sources/PocketKit/PocketNotificationService.swift

```diff
@@ -79,12 +100,37 @@ class PocketNotificationService: NSObject {
         }
     }
 
+    /**
+     Perform the following notification actions on Logout:
+     - Remove instant sync
+     */
+    private func loggedOut() {
```

Comment:
Do we need the complement to `braze.changeUser(userId:) in this function?

Reply:
That's part of the issue I'm emailing braze about, it doesn't exist atm.